app/views/shared/_navbar.html.erb

```diff
@@ -40,7 +40,7 @@
             <div class="dropdown-menu dropdown-menu-end nav-text" aria-labelledby="navbarDropdown">
               <%= link_to "My bookings", bookings_path, class: "dropdown-item" %>
 
-              <%= link_to "Account", "#", class: "dropdown-item border-bottom" %>
+              <%= link_to "Account", user_path(@user), class: "dropdown-item border-bottom" %>
```

Comment:
not working on the home page, because the home page doesn't have a user_id (so no current_user?). but works for other pages

Reply:
This should fix it

`class PagesController < ApplicationController
  skip_before_action :authenticate_user!, only: :home

  def home
    **@user = current_user**
  end
end
`